code/game/objects/items/weapons/clown_items.dm

```diff
@@ -17,11 +17,11 @@
  */
 /obj/item/weapon/soap/New()
 	..()
-	create_reagents(5)
+	create_reagents(15)

```

Comment:
because soap is near useless as of now should have been a separate PR but i edited the wrong branch then went 'oh fuck the web editor is useless'

Reply:
Why do you still use the web editor, then?

Reply:
very good question, i'll revert this change if wanted